apps/web/src/routes/orders/order-list/components/forms/edit-order-form.tsx

```diff
@@ -32,6 +34,6 @@ export const EditOrderForm: React.FC<{ order: IOrder; onClose: () => void }> = (
       }
     );
   };
-
-  return <OrderForm order={order} onSubmit={handleSubmit} onClose={onClose} isPending={isPending} />;
+  if (isLoading) return <OrderFormSkeleton />;
```

Comment:
super-nit: but I would love some breathing space between the lines, its not a part of the review but it feels more easier on the eyes

Reply:
Okay zain bhai, on it